experimental/tfhub/tests/test_tfhub.py

```diff
@@ -56,3 +82,27 @@ def test_ssd(spark: SparkSession):
     """
     )
     assert result.count() == 1
+
+
+def test_multi_pics_ssd(spark: SparkSession):
+    spark.udf.register("to_image2", to_image)
+    spark.sql(
+        f"""
+        CREATE MODEL tfssd2
+        MODEL_TYPE ssd
+        OPTIONS (device="cpu", batch_size=32)
+        USING "tfhub:///tensorflow/ssd_mobilenet_v2/2";
```

Comment:
Here is the corresponding TFHUB uri: https://tfhub.dev/tensorflow/ssd_mobilenet_v2/2